app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PasswordsController < ApplicationController
+  def edit
+  end
+
+  def update
+    @user = current_user
+    @user.update(user_params)
+    if @user.save
+      flash[:success] = "Your Password Has Been Updated"
+      redirect_to "/profile"
+    else
+      flash[:error] = "Please enter valid & matching passwords"
+      redirect_to "/password/edit"
```

Comment:
I feel like this is not ideal... it is just supposed to just render the page, but was not having good luck with the `render` call